metricbeat/module/mongodb/collstats/_meta/docs.asciidoc

```diff
@@ -0,0 +1 @@
+This is the `collstats` metricset of the module mongodb.
```

Comment:
Could you add a bit of information about what information or commands that it uses to collect the data from mongodb. This might help someone familiar with mongodb to quickly understand what the metricset does.